crates/cli/src/util.rs

```diff
@@ -46,7 +46,7 @@ pub async fn spacetime_dns(
 /// identity will be looked up in the config and it will be used instead. Returns Ok() if the
 /// domain is successfully registered, returns Err otherwise.
 pub async fn spacetime_register_tld(
-    config: &mut Config,
+    config: &Config,
```

Comment:
unrelated cleanup. I think this function used to manage identities in some way, but no longer needs a `&mut Config` since our identity management is more global now.